### ISSUE TITLE
Anchor name "Fully_Qualified_Name" used more than once

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#20512]
`Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once`

We need to fix this warning and make junit test to prevent this leak in future.

detected at:
https://github.com/checkstyle/checkstyle/actions/runs/28518105896/job/84536639224#step:5:231

```
[INFO] Rendering content with org.apache.maven.skins:maven-fluido-skin:jar:2.0.1 skin
[INFO] Rendering 282 Doxia documents: 282 xdoc
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
Warning:  src/site/xdoc/checks/coding/unusedtryresourceshouldbeunnamed.xml, line 119, column 74: Anchor name "Fully_Qualified_Name" used more than once
[INFO] Rendering 23 report documents
[INFO] Generating "Dependencies" report  --- maven-project-info-reports-plugin:3.9.0:dependencies

```

**Comment by Carbon14-48** [comment]
on it

**Comment by romani** [comment]
Fix is merged
